woocommerce/woocommerce-gutenberg-products-block
# Tag List block: Add support for global style

~~This PR is blocked by #5515~~

This PR adds support for global style for the `Tag List block`.

Now, the user can edit the style for:
- text-color
- link-color
- font-size


#4965 

### Screenshots

![image](https://user-images.githubusercontent.com/4463174/148409528-d63873fb-ceef-46dd-bb54-7fd08f2e3164.png)
*without global style*


![image](https://user-images.githubusercontent.com/4463174/148409568-df34e834-97a4-4502-818d-96cfde79e106.png)
*with global style*


### Testing

### Manual Testing

Check out this branch:

1. On `WordPress 5.9`, install and enable the `Gutenberg` plugin.
2. Install and enable the `Twenty Twenty-Two` theme.
3. Add the `Single Product block` (this block contains `Tag List block`) to a post.
4. Get the focus on `Tag List block`.
5. On the right sidebar, personalize the styles of the block.
6. Go on the page and check if there are changes.
7. Reset to default using the `Reset` button from the different sections. 
8. Go to Dashboard and select Appearance > Editor (beta). On top of the screen, select Home > Browser all templates > Single Post. When the page is loaded, add the block to the page.
9. On the Editor page click on the `Styles` icon on the right-top corner.
10. Verify that the `Tag List block` is shown under the `Blocks` section. Personalize again the block.
11. Save your changes.
12. Go on the page created earlier and check if all styles are applied correctly.
13. Edit your previous post/page again.
14. Change again the styles.
15. Save your changes.
16. Check if these styles have priority over the styles from the Site Editor.

### Changelog

> Add support for the global style for the Tag List block.
